cypress/integration/recipe_detail_spec.js

```diff
@@ -0,0 +1,354 @@
+        localStorage.setItem('inProgressRecipes', JSON.stringify(inProgressRecipes));
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').contains('Continuar Receita');
+  });
+
+  it('verifica botão de Continuar Receita na tela de detalhes de uma bebida', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        const inProgressRecipes = [178319];
+        localStorage.setItem('inProgressRecipes', JSON.stringify(inProgressRecipes));
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').contains('Continuar Receita');
+  });
+});
+
+describe('Quando o botão "Iniciar Receita" for clicado, a rota deve mudar para a tela de receita em processo', () => {
+  it('redireciona para tela de receita (de uma comida) em progresso', () => {
+    cy.visit('http://localhost:3000/comidas/52771', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').click();
+    cy.location('href').should('include', '/in-progress');
+  });
+
+  it('redireciona para tela de receita (de uma bebida) em progresso', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').click();
+    cy.location('href').should('include', '/in-progress');
+  });
+});
+
+// describe('Um botão de compartilhar e um de favoritar a receita devem estar disponíveis');
+
+// describe('Ao clicar no botão de compartilhar, o link da receita dentro do app deve ser copiado para o clipboard e uma mensagem avisando que o link foi copiado deve aparecer');
+
+// describe('O ícone do coração (favorito) deve vir preenchido caso a receita esteja favoritada, e _"despreenchido"_ caso contrário');
+
+// describe('Ao clicar no botão de favoritar, o ícone do coração deve mudar de seu estado atual, caso esteja preenchido deve mudar para _"despreenchido"_ e vice-versa');
+
+describe('As receitas favoritas devem ser salvas em `localStorage` na chave `favoriteRecipes`', () => {
+  it('favorita receita de uma comida', () => {
+    cy.visit('http://localhost:3000/comidas/52771', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="favorite-btn"]').click().then(() => {
+      const favoriteRecipes = JSON.parse(localStorage.getItem('favoriteRecipes'));
+      expect(favoriteRecipes.some(f => f.id == 52771)).to.be.true;
+    });
+  });
+
+  it('favorita receita de uma bebida', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="favorite-btn"]').click().then(() => {
+      const favoriteRecipes = JSON.parse(localStorage.getItem('favoriteRecipes'));
+      expect(favoriteRecipes.some(f => f.id == 178319)).to.be.true;
```

Comment:
A mesma coisa aqui sobre ser mais exato na comparação e sobre limpar o `localStorage` após o teste.

```suggestion
      const expectedFavoriteRecipes = [
        {
          id: 178319,
          type: 'bebida',
          area: '',
          category: 'Cocktail',
          alcoholicOrNot:  'Alcoholic',
          name: 'Aquamarine',
          image: 'https://www.thecocktaildb.com/images/media/drink/zvsre31572902738.jpg',
        },
      ];

      expect(favoriteRecipes).to.deep.eq(expectedFavoriteRecipes);
      localStorage.clear();
```